packages/wdio-junit-reporter/src/types.ts

```diff
@@ -49,6 +55,15 @@ export interface JUnitReporterOptions extends Reporters.Options {
      * }
      */
     classNameFormat?: (options: ClassNameFormatOptions) => string
+    /**
+     * Give the ability to override the generated name of a test suite. suiteNameFormat RegExp would be applied to the suite name for normalization.
+     *
+     * @default
+     * testSuiteNameFormatOptions: function (options) {
```

Comment:
```suggestion
     * suiteNameFormat: function (options) {
```

Reply:
testSuiteNameFormatOptions is already in use